dxvk.conf

```diff
@@ -557,3 +546,17 @@
 # - True/False
 
 # dxvk.enableDebugUtils = False
+
+# Texture unmapping delay
+#
+# DXVK will unmap D3D9 texture data after a certain number of frames.
+# 0 to disable unmapping.
+
+# d3d9.textureUnmapDelay = 16
+
+# Buffer unmapping delay
+#
+# DXVK will unmap D3D9 buffer data after a certain number of frames.
+# 0 to disable unmapping.
+
+# d3d9.bufferUnmapDelay = 16
```

Comment:
`this->bufferUnmapDelay              = config.getOption<int32_t>     ("d3d9.bufferUnmapDelay",              256);`

Also, will e.g. `d3d9.presentInterval = 2` affect those delays? I didn't read the code, but delays will be 8 and 128 real frames in this case.

Reply:
> Also, will e.g. d3d9.presentInterval = 2 affect those delays? I didn't read the code, but delays will be 8 and 128 real frames in this case.

It should but that also shouldn't be a problem. This is just the first best thing I came up with and the values are somewhat arbitrary.